drivers/serial/uart_mchp_mec5.c

```diff
@@ -0,0 +1,639 @@
+/*
+ * Copyright (c) 2024 Microchip Technology Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @brief Microchip MEC5 ns16550 compatible UART Serial Driver
+ */
+
+#define DT_DRV_COMPAT microchip_mec5_uart
+
+#include <errno.h>
+#include "zephyr/devicetree.h"
```

Comment:
Probably a good idea to change from local include to global include.

Reply:
@dcpleung  Done and pushed update